packages/rollup-core/src/app/constants.ts

```diff
@@ -44,3 +44,32 @@ export const L2_TO_L1_MESSAGE_PASSER_OVM_ADDRESS =
 
 // See the getTransactionBatchCalldata(...) function of canonical-chain-batch-submitter.ts for more info
 export const ROLLUP_TX_SIZE_IN_BYTES_MINUS_CALLDATA = 150
+
+/*
+Static bytes: 160
+- 4 (Method ID)
+- 32 (txs byte start)
+- 32 (timestamp)
+- 32 (block number)
+- 32 (starts at index)
+- 32 (number of txs bytes elements)
+ */
+export const L1_ROLLUP_BATCH_TX_STATIC_CALLDATA_BYTES = 164
+
+/*
+- 32 for start of each `bytes`
+- 32 for length of each `bytes`
+ */
+export const L1_ROLLUP_BATCH_TX_BYTES_PER_L2_TX = 64
+
+/*
+- 32 for nonce
+- 32 for gas price
+- 32 for gas limit
+- 32 for value
+- 32 for R (signature)
+- 32 for S (signature)
+- 2 (at most) for V (signature)
```

Comment:
Shouldn't this always be two, ie padded to 2 bytes if it is smaller?

Reply:
Ahh, so this is the L1 tx, so we won't be padding it like the L2 ones. I could be wrong, but I think V is only 1 byte for mainnet txs because Chain ID is < 255. We only needed to worry about padding for our L2 Chain ID of 420 since it requires more than 1 byte but is able to be represented with 3 hex chars instead of requiring all 4. Either way, we should be fine if we just assume V always takes 2 bytes -- our goal is to not underestimate. Being over by 1 byte worst case should be fine.